kustomize/overlays/minikube/shard/patches/deploy-config.yaml

```diff
@@ -5,6 +5,7 @@ data:
   EVENT_BRIDGE_K8S_PLATFORM: k8s
   EVENT_BRIDGE_MANAGER_URL: http://event-bridge.event-bridge-manager.svc.cluster.local:8080
   EVENT_BRIDGE_SSO_URL: http://<REPLACE_WITH_MINIKUBE_IP>:30007/auth/realms/event-bridge-fm
+  EVENT_BRIDGE_SHARD_ID: "55095146"
```

Comment:
`minikube` will use the local keycloak instance. Replace this with https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/ef4d9bb32b4ab05e1faf682a4664c09f48f4e3fd/kustomize/overlays/minikube/keycloak/event-bridge-fm-realm-sample.json#L43

Reply:
done